### PR TITLE
Add support for AquastreamXT pump

### PR DESCRIPTION
**Pull request by leoratte**
Add support for the Aquacomputer AquastreamXT pump. Currently only the temperature sensors are working.

### CODE REVIEW

File: aquacomputer_d5next.c [hunk 1]
```diff
@@ -215,6 +221,13 @@ static u8 leakshield_usb_report_template[] = {
 #define LEAKSHIELD_USB_REPORT_UNIT_RPM		0x03
 #define LEAKSHIELD_USB_REPORT_UNIT_DL_PER_H	0x0C
 
+/* Register offsets for */
```
Comment (aleksamagicka) [inline aquacomputer_d5next.c:224]: Unfinished comment.

File: aquacomputer_d5next.c [hunk 2]
```diff
@@ -362,6 +375,13 @@ static const char *const label_leakshield_fan_speed[] = {
 	"Reservoir Filled [ml]",
 };
 
+/* Labels for AquastreamXT */
+static const char *const label_temp_sensors_aquastreamxt[] = {
+	"Fan ctrl temp",
+	"Ext Temp",
```
Comment (aleksamagicka) [inline aquacomputer_d5next.c:381]: Perhaps call this `External sensor`, that's how it's called for the High Flow Next.

File: aquacomputer_d5next.c [hunk 3]
```diff
@@ -362,6 +375,13 @@ static const char *const label_leakshield_fan_speed[] = {
 	"Reservoir Filled [ml]",
 };
 
+/* Labels for AquastreamXT */
+static const char *const label_temp_sensors_aquastreamxt[] = {
+	"Fan ctrl temp",
```
Comment (aleksamagicka) [inline aquacomputer_d5next.c:380]: Just for my reference, if I'm understanding correctly this is the temp that the pump can use to control the optionally attached fans?

Comment (leoratte) [reply aquacomputer_d5next.c:380]: No, this is the temp of the IC controlling the fan voltage.

**Comment by aleksamagicka** [comment]
Thanks for the quick turnaround, much appreciated.

**Pull request merged**